index.html

```diff
@@ -6,13 +6,26 @@
 	-->
 	<head>
 
-		<title><!-- your title goes here --></title>
+		<title>
+			<center>My Two Texts</center>
+			
+		</title>
 
 		<style>
 			body {
 				padding: 0;
 				margin: 0;
+				margin-left:30%
+				margin-right:30%
```

Comment:
Missing a semicolon (`;`) after your CSS properties here, which is causing issues.